src/olympia/users/templates/users/login.html

```diff
@@ -15,7 +15,13 @@
   {% else %}
     {% include "users/login_help.html" %}
     <section class="island hero primary grid prettyform">
-      <h1>{{ _('Enter your email') }}</h1>
+      <h1>{{ _('Enter your email to log in or register') }}</h1>
+      <p class="island-note">
+        {% trans %}
+          As the Firefox Accounts migration continues we will detect if you
```

Comment:
A user might not realise it has ever started. Also "log in or register on this site or on Firefox Accounts" is hard to parse. Maybe:

"We are migrating existing Add-ons users to Firefox accounts. We will detect if your account needs migrating automatically."